docs/usage/faq_and_troubleshooting.md

```diff
@@ -291,32 +290,6 @@ The `work/` directory can be hard to navigate due to the `<short_hash>/<long_has
 Otherwise, you can use a utility like [tree](<https://en.wikipedia.org/wiki/Tree_(command)>) to show the directory
 structure, which allows you to manually find the target process directory.
 
-## Saving logs from the `work/` directory
-
```

Comment:
you don't need this section anymore?

Reply:
That's right! This has been implemented at the module level and documentation moved to docs/outputs.md